typings/chai-immutable.d.ts

```diff
@@ -1,5 +1,7 @@
 declare module "chai-immutable" {
 	function chaiImmutable(chai: any, utils: any): void;
-	namespace chaiImmutable { }
+	namespace chaiImmutable {
+
+	}
```

Comment:
I agree, but I think the solution is to use a more complete type declaration no?

Reply:
Creating definition files is quite a pain in my experience, so I tend to start with just minimum required to get things compiling.  In this case, since it is for testing I wasn't too strict on the definition but I needed something that would result in compilation not failing.

The actual usage is only this:
```typescript
import * as chaiImmutable from 'chai-immutable'
chaiUse(chaiImmutable)
```

If you remove any single line in that file then that will fail, but I don't actually use any of the type definitions anywhere.  If you have a better way to make it so `chaiUse` will accept `chaiImmutable` I'm open to ideas.  😁